component/service/src/main/java/org/exoplatform/social/rest/api/EntityBuilder.java

```diff
@@ -192,9 +192,12 @@ public static ProfileEntity buildEntityProfile(Profile profile, String restPath,
       User user = organizationService.getUserHandler().findUserByName(userEntity.getUsername());
       if (user != null) {
         userEntity.setIsInternal(user.isInternalStore());
+        if (!user.getLastLoginTime().equals(user.getCreatedDate())) {
+          userEntity.setLastConnexion(String.valueOf(user.getLastLoginTime().getTime()));
+        }
       }
     } catch (Exception e) {
-      LOG.warn("Error while checking internal store user", e);
+      LOG.warn("Error when searching user " + userEntity.getUsername(), e);
```

Comment:
Use Message template better than string concatenation in logs

```suggestion
      LOG.warn("Error when searching user {}", userEntity.getUsername(), e);
```